aws-blog-event-driven-batch-analytics/State-Management-Store.md

```diff
@@ -0,0 +1,45 @@
+
+
+## State Management Store
```

Comment:
Can we add a preface here about what the purpose of the state management store is, and how it's implemented in the solution (mysql RDS, etc).